packaging/debian/install

```diff
@@ -1,7 +1,12 @@
-etc/apt/apt.conf.d/*
-etc/modprobe.d/*
-etc/rsyslog.d/*
-etc/sysctl.d/*
-lib/udev/rules.d/*
-lib/udev/*
-usr/bin/*
+# Upstream-provided configuration
+src/etc/apt/apt.conf.d/* etc/apt/apt.conf.d
+src/etc/modprobe.d/* etc/modprobe.d
+src/etc/rsyslog.d/* etc/rsyslog.d
+src/etc/sysctl.d/* etc/sysctl.d
+src/lib/udev/* lib/udev
+src/usr/bin/* usr/bin
+
+# Ubuntu-specific configuration
```

Comment:
Unless I'm missing something, this is installing Ubuntu specific stuff everywhere now (including Debian). Some of these settings (`instance_configs.cfg.distro` in particular) are definitely not appropriate for Debian.

Reply:
I like the `packaging/ubuntu` approach, this way we can maintain the packaging together and work to unify our irqbalance approach.

No need to restore Stretch and Jessie conflicts though, those are fine to drop.

Reply:
> I like the packaging/ubuntu approach, this way we can maintain the packaging together and work to unify our irqbalance approach.

Perfect, I reworked this PR to do that.